CHANGELOG.md

```diff
@@ -1,5 +1,173 @@
 # Changelog
 
+## v2.6.0 - 2024-04-30
+
+This minor release comes with a number of features that are focused on user friendliness and ease-of-use of the CLI and the API.
+The caching mechanism has received a number of improvements guaranteeing even greater savings of computational time, but this requires a database migration.
+Therefore, when updating your installation, you will be prompted to update your database, which is fully automatic.
```

Comment:
The migration itself is automatic, but users need to initiate rehash themselves (at least that's how it worked in previous releases AFAIK). If that's the case, would be good to mention that here.

Reply:
I see you mention this in the dedicated section below, but I'd still point it out here as well.